Formula/vault.rb

```diff
@@ -28,15 +28,10 @@ class Vault < Formula
 
   depends_on "go" => :build
   depends_on "node@18" => :build
-  depends_on "python@3.11" => :build
   depends_on "yarn" => :build
 
   def install
-    # Needs both `npm` and `python` in PATH
-    ENV.prepend_path "PATH", Formula["node@18"].opt_libexec/"bin"
-    ENV.prepend_path "PATH", "#{ENV["GOPATH"]}/bin"
-    ENV["PYTHON"] = "python3.11"
-    system "make", "bootstrap", "static-dist", "dev-ui"
+    system "make", "static-dist", "dev-ui"
```

Comment:
`bootstrap` target no longer needed?

Reply:
it downloads some go tools seemingly only needed for development: https://github.com/hashicorp/vault#developing-vault. but put this back in case it changes in the future